readthedocs_build/config/test_find.py

```diff
@@ -75,3 +75,18 @@ def test_find_multiple_files(tmpdir):
         str(tmpdir.join('first', 'readthedocs.yml')),
         str(tmpdir.join('third', 'readthedocs.yml')),
     ]
+
+
+def test_find_unicode_path(tmpdir):
+    base_path = os.path.abspath('integration_tests/bad_encode_project')
+    try:
+        path = find_one(base_path, ('readthedocs.yml',))
+        # When python3
+        assert path == ''
+    except Exception as e:
+        # When python2
+        assert isinstance(e, UnicodeDecodeError)
```

Comment:
Instead of a try/except here, I'd use six or basic python version_info checking to test for these two cases:

```python
sys.version_info < (3,0,0)
```